en-US/Resources.xml

```diff
@@ -52,6 +52,44 @@
 				<para>
 					<citetitle>IBM Style Guide</citetitle>. IBM Corporation, latest version.
 				</para>
+				<para>
+					Red Hat employees can now access the latest version of IBM Style online, at the following site: 
```

Comment:
Missing non-breaking space, and I don't think "IBM Style" is the correct title.
Do you need to include "at the following site" and then put the url in a separate para? How about just "at <url>"?

Reply:
The title in the PDF is "IBM Style Guide", while on the online site it is "IBM Style". 
I will continue to apply the PDF title for now; we might need to revisit in future if the PDF is discontinued.